bin/plugin/commands/since.js

```diff
@@ -17,6 +18,10 @@ const { log, formats } = require( '../lib/logger' );
  */
 
 exports.options = [
+	{
+		argname: '-p, --plugin <plugin>',
+		description: 'Plugin name',
```

Comment:
@felixarntz Any specific reason to add this, since by default it will run on the PL plugin anyway?

Reply:
@thelovekesh From an API perspective it's better to have an actual value defined. We shouldn't use a default that is impossible to manually pass IMO.

This also creates a consistent DX. No matter which plugin you run the command for, you can pass the plugin slug via `--plugin`. The only reason that you don't _have_ to do this for `performance-lab` is backward compatibility.

Last but not least, having a distinct value for `performance-lab` will help later if we decide to move that plugin's code to `./plugins/performance-lab` and therefore treat it similar to the other plugins.

Reply:
Humm, I see this change in the readme command as well.

Reply:
How do you mean? The readme command checks the provided plugin slug against the milestones, which means `performance-lab ...` would also be covered. But since it's not in `plugins.json`, here we have to manually check for it.

Reply:
By this change, I meant using performance-lab check for plugin root.